fusion/fusion.py

```diff
@@ -958,10 +962,10 @@ def puzzle_for_coin(coin: Coin) -> Program:
 
 
 async def select_coins(wallet_client: WalletRpcClient, amount: uint64):
-    excluded_coins=list()
-    coin_selection_config = CoinSelectionConfig(min_coin_amount=1, max_coin_amount=9999999999999, excluded_coin_amounts=[], excluded_coin_ids=[])
+    excluded_coin_ids = [c.name() for c in recent_coins]
```

Comment:
@BrandtH22 FYI, found this code I thought I already had...this is to track coins used for minting singletons (could potentially create a shared model to track such things across all components)